src/tests/SharpRaven.UnitTests/Data/Context/AppTests.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Newtonsoft.Json;
+using NUnit.Framework;
+using SharpRaven.Data.Context;
+
+namespace SharpRaven.UnitTests.Data.Context
+{
+    [TestFixture]
+    public class AppTests
+    {
+        [Test]
+        public void Create_StartTime_SometimeInThePast()
+        {
+            var sut = App.Create();
+
+            Assert.True(sut.StartTime.HasValue);
+            Assert.AreNotEqual(default(DateTimeOffset), sut.StartTime.Value);
+            Assert.Greater(DateTimeOffset.UtcNow, sut.StartTime);
```

Comment:
I think `Assert.That(app.StartTime, Is.GreaterThan(DateTimeOffset.UtcNow))` reads better.